recipes/mesa/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "11.2.2" %}
+
+package:
+  name: osmesa
+  version: {{ version }}
+
+source:
+  url: https://github.com/mesa3d/mesa/archive/mesa-{{ version }}.tar.gz
+  fn: mesa-{{ version }}.tar.gz
+  sha256: a2edf7ad2589f0c3b5cd3d8f8866193c03bf871a38d5dc27423a03f352abe2de
+
+build:
+  number: 1
+  msvc_compiler: 14.0
+
+requirements:
+  build:
+    # requirements for Mesa's SCons scripts
+    - python 2.7.*
+    - scons
+    - mako
+    # requirements for compiling Mesa
+    - toolchain
+    - pkg-config
+    - autoconf
+    - automake
+    - libtool
+    - m4 # [unix]
+    - flex  # [unix]
+    - bison  # [unix]
+    - mingwpy  # [win]
```

Comment:
You can remove this if not building on Windows.
Also most of the remaining dependencies are Linux (and maybe OS X) only, so would not bother adding the `[unix]` pre-selector there.